controller/integration/wrapper/drainerconfig/wrapper.go

```diff
@@ -114,15 +84,17 @@ func New(config Config) (*Wrapper, error) {
 	{
 		c := controller.Config{
 			CRD:       v1alpha1.NewDrainerConfigCRD(),
-			CRDClient: crdClient,
-			Informer:  newInformer,
+			K8sClient: k8sClient,
 			Logger:    config.Logger,
 			ResourceSets: []*controller.ResourceSet{
 				resourceSet,
 			},
-			RESTClient: g8sClient.CoreV1alpha1().RESTClient(),
+			RuntimeObjectFactory: func() pkgruntime.Object {
+				return &v1alpha1.DrainerConfig{}
```

Comment:
```suggestion
				return new(v1alpha1.DrainerConfig)
```
Same as above. Just a candidate in case you like this :slightly_smiling_face: